src/util/queryKeys.tsx

```diff
@@ -2,6 +2,7 @@ export const queryKeys = {
   certificates: "certificates",
   cluster: "cluster",
   configOptions: "configOptions",
+  forwards: "forwards",
```

Comment:
I think `networkForwards` is a bit clearer?

Reply:
I was mimicking the api path, so this key is always used in combination with networks, i.e. `queryKey: [queryKeys.networks, network, queryKeys.forwards, project],`.

Reply:
I see, thanks for clarifying!